sdk/storage/azure-storage-file/tests/test_file.py

```diff
@@ -263,7 +263,7 @@ def test_create_file_with_metadata(self):
 
     def test_create_file_when_file_permission_is_too_long(self):
         file_client = self._get_file_client()
-        permission = str(self.get_random_bytes(8 * 1024 + 1))
+        permission = str(self.get_random_bytes(8 * 1024))
```

Comment:
Why was this removed? Looks like part of the test.

Reply:
oops...updated it back.. :P